Content.Shared/Wieldable/WieldableSystem.cs

```diff
@@ -155,7 +154,7 @@ public bool CanWield(EntityUid uid, WieldableComponent component, EntityUid user
             return false;
         }
 
-        if (hands.CountFreeHands() < component.FreeHandsRequired)
+        if (hands.Count < component.FreeHandsRequired)
```

Comment:
This should probably use a new method that checks for CountFreeableHands or something.

Also the `wieldable-component-not-enough-free-hands` string needs to be changed.

Reply:
What should it be changed to? I think it still conveys the meaning of "you can't drop this so you don't have enough free hands"
```
wieldable-component-not-enough-free-hands = {$number ->
    [one] You need a free hand to wield { THE($item) }.
    *[other] You need { $number } free hands to wield { THE($item) }.
}
```

Reply:
First thought yesterday was to keep with the naming `wieldable-component-not-enough-freeable-hands`, but I'm not sure how readable that is to localizers.

Alternatives `wieldable-component-not-enough-available-hands` or `wieldable-component-not-enough-usable-hands`

Reply:
> First thought yesterday was to keep with the naming `wieldable-component-not-enough-freeable-hands`, but I'm not sure how readable that is to localizers.
> 
> Alternatives `wieldable-component-not-enough-available-hands` or `wieldable-component-not-enough-usable-hands`

I think its better to keep the id the same since it would be such a minor change from the original, just to not break any existing localization of it, but I could change if you think that's not a big deal